proposals/xxxx-continous-integration.md

```diff
@@ -0,0 +1,54 @@
+# Configuration management
+
+* Proposal: [SDL-xxxx](xxxx-Configuration-Management.md)
+* Author: [Vadym Kushnirenko](https://github.com/vkushnirenko-luxoft)
+* Status: **Awaiting review**
+* Impacted Platforms: [Core/??]
+
+## Introduction
+
+This proposal is about configuration management system for continuous building and testing code.
+
+## Motivation
+
+The development process should be open and transparent. Any person could deploy it on his own side, use it and propose recomendations about improvements.
+Configuration must be saved as a code, be flexible and be renewal in any moment from public resources.
+
+## Proposed solution
+
+The solution proposes to use docker containers for deploying all components. For building and keeping artifacts we must use next tools:
+1) Jenkins - continuous integration system.
+2) Docker containers - to up clean build environment quickly and scalable.
+3) Github - to keep Dockerfiles and Jenkins jobs.
+4) Artifactory - to keep artifacts - builds and test results.
+
+### Github
+Create a repository to save code of environment. This repository includes:
```

Comment:
@vkushnirenko-luxoft 
Not quite clear what is ".. code of environment". I think it should be ".. to store build environment configurations"